src/NServiceBus.Core/IMessageSessionExtensions.cs

```diff
@@ -78,7 +78,7 @@ public static Task Send<T>(this IMessageSession session, string destination, Act
         }
 
         /// <summary>
-        /// Sends the message back to the current endpoint.
+        /// Sends the message back to the current endpoint. Shortcut for `SendOptions.RouteToThisEndpoint()`.
```

Comment:
I would prefer this being an actual xml reference to the extension method `RouteToThisEndpoint`